applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationAsyncClient.java

```diff
@@ -117,6 +157,24 @@ public static ConfigurationAsyncClientBuilder builder() {
      * setting's etag matches. If the etag's value is equal to the wildcard character ({@code "*"}), the setting
      * will always be updated.
      *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * <pre>
+     * ConfigurationSetting setting = new ConfigurationSetting().key("prodDBConnection")
+     *     .label("westUS")
+     *     .value("db_connection");
+     *
+     * client.setSetting(setting).subscribe(response -&gt; {
+     *     ConfigurationSetting result = response.value();
+     *     System.out.printf("Key: %s, Value: %s", result.key(), result.value());
+     * });
+     *
+     * setting = setting.value("updated_db_connection");
```

Comment:
This is not good code - ConfigurationSetting is a mutable class, so you do not need to reassign it to `setting` again